### PR TITLE
Browse Page - Sort

### PR DESCRIPTION
**Pull request by slimonslimon**
There is an implementation of an other usefull function on browse page.

Sorting is done by sqlite queries, so results can be sorted only by colums which are saved in DB. We added a new colum with Datetime of start of flight.

But there is an Python error -> I really dont know, how to call Super class init metod coreclty with multiple parents..

### CODE REVIEW

File: tornado_handlers/browse.py [hunk 1]
```diff
@@ -57,51 +88,75 @@ def get_columns_from_tuple(db_tuple, counter):
...
-                return None
+            generateddata_log_id = db_tuple[6]
+            if log_id != generateddata_log_id:
+                print('Join failed, loading and updating data')
+                db_data_gen = get_generated_db_data_from_log(log_id, con, cur)
+                if db_data_gen is None:
+                    return None
+                db_data.add_generated_db_data_from_log(db_data_gen)
+            else:
+                db_data.duration_s = db_tuple[7]#7
```
Comment (bkueng) [inline tornado_handlers/browse.py]: Can you remove the `#7`?

**Comment by bkueng** [comment]
@slimonslimon ping?

File: plot_app/db_entry.py [hunk 2]
```diff
@@ -123,6 +124,21 @@ def from_log_file(cls, log_id):
...
 
+        # logging start time & date
+        try:
+           # get the first non-zero timestamp
+            gps_data = ulog.get_dataset('vehicle_gps_position')
+            indices = np.nonzero(gps_data.data['time_utc_usec'])
+            if len(indices[0]) > 0:
+                # we use the timestamp from the log and then convert it with JS to
+                # display with local timezone.
+                # In addition we add a tooltip to show the timezone from the log
```
[the remark this answers is not in the document]
Comment (slimonslimon) [reply plot_app/db_entry.py]: ok - deleted

**Comment by slimonslimon** [comment]
thank you for your comments and advices.

I still getting folownig message when running tests (and I dont know what can I do with this)
tornado_handlers/browse.py:29:4: R0915: Too many statements (114/100) (too-many-statements)

I have tried to recreate first column of table to non-orderable "row number" - is it correct?

**Comment by bkueng** [comment]
> But Im able to create same number by "cross product-where-group by-count" construction. (of course qudratic number of row are processed)

I'd use the previous version then, I'd like to keep the query efficient.

**Comment by slimonslimon** [comment]
So sould I revert commit: https://github.com/PX4/flight_review/pull/151/commits/674db00f28c3d771ef69438f4ca18dbcd8168afa ?

**Comment by slimonslimon** [comment]
Okey.

**Pull request merged**